source/textInfos/__init__.py

```diff
@@ -551,6 +553,15 @@ def getMathMl(self, field):
 		"""
 		raise NotImplementedError
 
+	def isOutOfBounds(self):

```

Comment:
I'm not sure whether I think this method name is clear enough. isOffscreen or isInvisible might sound clearer. Out of bounds becomes ambiguous if we implement more bounds, i.e. bounds of screen, bounds of the foreground window, bounds of a given location object, etc.